components/pre_trial_pricing_modal.tsx/content.tsx

```diff
@@ -0,0 +1,293 @@
+            return state.entities.cloud.products[subscription?.product_id];
+        }
+        return undefined;
+    });
+
+    let isStarter = false;
+    if (product?.sku === CloudProducts.STARTER) {
+        isStarter = true;
+    }
+    return (
+        <div className='Content'>
+            <div className='self-hosted-alert'>
+                <h1>{formatMessage({id: 'pretrial_pricing_modal.lookingToSelfHost', defaultMessage: 'Looking to self-host?'})}</h1>
+                <a
+                    onClick={() =>
+                        trackEvent(
+                            TELEMETRY_CATEGORIES.CLOUD_PURCHASING,
+                            'click_looking_to_self_host',
+                        )
+                    }
+                    href={CloudLinks.DEPLOYMENT_OPTIONS}
+                    rel='noopener noreferrer'
+                    target='_blank'
+                >{formatMessage({id: 'pretrial_pricing_modal.reviewDeploymentOptions', defaultMessage: 'Review deployment options'})}</a>
+            </div>
+            <Modal.Header className='PreTrialPricingModal__header'>
+                <h1 className='title'>
+                    {formatMessage({id: 'pretrial_pricing_modal.title', defaultMessage: 'Select a plan'})}
+                </h1>
+                <button
+                    id='closeIcon'
+                    className='icon icon-close'
+                    aria-label='Close'
+                    title='Close'
+                    onClick={() => {}}
+                />
+            </Modal.Header>
+            <Modal.Body className='PreTrialPricingModal__body'>
+                <div className='self-hosted-svg-left'>
+                    <LadySvg/>
+                </div>
+                <div className='self-hosted-svg-right'>
+                    <ManSvg/>
+                </div>
+                <Card
+                    topColor='#9DA7B8'
+                    plan='Starter'
+                    price={formatMessage({id: 'pretrial_pricing_modal.price.free', defaultMessage: 'Free'})}
+                    briefing={{
+                        title: formatMessage({id: 'pretrial_pricing_modal.briefing.starter', defaultMessage: 'Starter comes with...'}),
+                        items: [
+                            formatMessage({id: 'pretrial_pricing_modal.briefing.storage', defaultMessage: '{storage}GB file storage'}, {storage: '50'}),
+                            formatMessage({id: 'pretrial_pricing_modal.briefing.starter.messageHistory', defaultMessage: 'Limited message history'}),
+                            formatMessage({id: 'pretrial_pricing_modal.briefing.starter.boardCards', defaultMessage: '{boards} Boards cards per server'}, {boards: '500'}),
+                            formatMessage({id: 'pretrial_pricing_modal.briefing.starter.teamCount', defaultMessage: 'Limited to one team'})],
+                    }}
+                    buttonDetails={{
+                        action: () => {},
```

Comment:
Should the button actions be implemented as part of this PR?

Reply:
@neallred some of the actions lead to another UI. For example, on the professional card, it opens a form to purchase and that screen is not implemented yet.

Reply:
We can have the action open the current purchase modal which is the one we will just update I believe

Reply:
It looks almost similar in terms of design, but we have to rewrite some of the logic.